doc/source/cluster/kubernetes.rst

```diff
@@ -292,6 +297,79 @@ To delete a running Ray cluster, you can run the following command:
 
   kubectl delete -f ray/doc/kubernetes/ray-cluster.yaml
 
+.. _k8s-gpus:
+
+Using GPUs
+----------
+
+Using GPUs on Kubernetes requires some setup.
+Here's where to find the relevant documentation for GPU usage on `GKE`_, for `EKS`_, and for `AKS`_.
```

Comment:
```suggestion
To use GPUs on Kubernetes, you will need to configure both your GKE setup and add additional values to your Ray cluster configuration.

For relevant documentation for GPU usage on different clouds, see instructions for `GKE`_, for `EKS`_, and for `AKS`_.
```

Reply:
Ah actually I guess it should say "Kubernetes setup" rather "GKE setup" (we should probably be cloud-agnostic)